### DISCUSSION TITLE
Install extensions (like PostGIS)

### DISCUSSION THREAD

**Discussion by development-vargamarcel** [nhost/nhost#38] [Ideas]
I would be useful to be able to install some critical extensions as postgis.

**Comment by taylormcknight** [comment]
Is there a way to do this currently or a workaround? Not being able to install postgis is the first roadblock we've run into using nhost and pretty mission-critical for us

**Reply by elitan** [reply]
It's now available for all new projects. Please reach out to me directly if you want me to activate it on a current Nhost project. 

But I have a feeling we've already talked and solved this for your Nhost project :)

**Comment by elitan** [comment]
We're working on this here https://github.com/nhost/postgres which will have the PostGIS extensions installed.

Going to be released in the coming days.

**Reply by elitan** [reply]
The new Postgres image is now in use. All new projects use this new image with Postgis available.

Here's a good guide to get started with Hasura and Postgis: [GraphQL and geolocation on Postgres (PostGIS using Hasura)](https://hasura.io/blog/graphql-and-geo-location-on-postgres-using-hasura-562e7bd47a2f/)

For existing project, please reach out to me if you want to upgrade to this new image with Postgis available. Mail: johan@nhost.io or via our [Discord](https://discord.com/invite/9V7Qb2U).

**Comment by bangdragon** [comment]
Hi all, how to enable postgis in Nhost.
